README.md

```diff
@@ -22,45 +29,89 @@ preview functionality in VSCode.
 
 ![](doc/img/reveal-md_code-plugin.png)
 
+### Live-preview of slides via VSCode plugin
```

Comment:
Can we seperate GIT stuff from VS stuff. Like this:

### Get repository data
To work on a local version of the presention, simply do:
1. Clone this repository, to re-use existing contents in the default branch:
```{bash}
git clone git@github.com:opengisch/talks_revealmd.git
```
2. Browse relevant existing content in the [`slides`](slides) folder.
3. Optionally create a new local branch named after your new presentation;
   especially if you already know that the content will be relevant for re-use.
```{bash}
git checkout -b <my-example-presentation>
```
6. If slide material is interesting for others, you can publish your local
   presentation into this online repository. If you wish that your presentation
   is available on the [**landing
   page**](https://opengisch.github.io/talks_revealmd/) throught automated
   rending via GitHub actions, then make a pull request into the master branch
   on GitHub.
   
### Work with VSCode

The easiest way is to locally draft presentations in markdown and use the
preview functionality in VSCode.

#### Live-preview of slides via VSCode plugin

The `VSCode Reveal` plugin supports live-preview of the slides. You can
just click on the plugin on the left bar, which features a slide overview and
currently four toolbar buttons at the top. If you split the editor right and
click on `"Revealjs: Show presentation by side"`, then you can instantly
navigate through the current snapshot of your presentation.


#### Browser presentation modus

There is also a browser presentation modus, which is triggered by the
`<Revealjs: Open presentation in browser>` icon (vertical up-pointing arrow).

[...]